README.md

```diff
@@ -100,11 +100,10 @@ New-Item -Path .git/hooks/pre-commit -ItemType HardLink -Value <absolute_path/to
 We use `mdbook-linkcheck` to validate URLs included in our documentation. To perform link checks, uncomment the `[output.linkcheck]` field in the `book.toml` configuration file and install `mdbook-linkcheck` with:
 
 ```bash
-> cargo install mdbook-linkcheck --git https://github.com/Michael-F-Bryan/mdbook-linkcheck --rev 14441d77646d58cea8ffc32fde9ea33b2bedb1a2
+> cargo install mdbook-linkcheck
 ```
 
-Note that we use an alpha version of `mdbook-linkcheck` to be able to use a feature that hasn't landed in a release yet.
-You will also need `mdbook` version `>= 0.3.5`.
+You will need `mdbook` version `>= 0.3.5` and `mdbook-linkcheck` version `>= 0.7.1` to check links.
```

Comment:
This was here before, but are we sure it still works on `mdbook 0.3.5`? Maybe we should just say "you will need a recent version".

Reply:
Sounds good to me.

Reply:
We could also just update the number to the most recent version...

Reply:
But then it will become out of date :)